unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Transform/Serialization/ECSTransformSerialization.cs

```diff
@@ -0,0 +1,57 @@
+using System.IO;
+using BinaryWriter = KernelCommunication.BinaryWriter;
+
+namespace DCL.ECSComponents
+{
+    public static class ECSTransformSerialization
```

Comment:
I'm a bit confused, is this not going to be a protobuff in the future?

Reply:
we are not going to use protobuf for transform component because it faster this way and it is the component that is updated more frequently